src/js/components/notification.jsx

```diff
@@ -35,10 +39,15 @@ export class NotificationComponent extends Component {
 
     render() {
         return (
-            <div key='content' className='sk-notification' onClick={ this.props.actions.hideSettingsNotification }>
+            <div key='content'
+                 className='sk-notification'
+                 onClick={ this.props.actions.hideSettingsNotification }>
                 <p>
-                    <span ref='text' dangerouslySetInnerHTML={ createMarkup(this.props.ui.text.settingsNotificationText) }></span>
-                    <a href='#' className='sk-notification-close' onClick={ this.props.actions.hideSettingsNotification }>&times;</a>
+                    <span ref='text'
+                          dangerouslySetInnerHTML={ createMarkup(this.props.ui.text.settingsNotificationText) }></span>
```

Comment:
I'm not sure I like this createMarkup helper. It's only hiding the `{ __html: html }` crap which is supposed to be ugly by design.